### ISSUE TITLE
osx64 incorrect nw

### ISSUE THREAD

**Issue by mattstyles** [mllrsohn/node-webkit-builder#176]
Actually, I probably shouldnt take a guess at the exact problem as I'm not sure but running `gulp` inside the examples directory currently does not work on a retina MBP running yosemite (10.10.1).

There are a couple of small issues. Firstly just running gulp hangs after the download, so I cut back to just running for an osx64 build, which also hangs after download. Once cached you can run it again but the build wont be valid with these errors:

```
Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @executable_path/../Frameworks/node-webkit Framework.framework/node-webkit Framework
  Referenced from: /Users/USER/*/nw-demo.app/Contents/MacOS/node-webkit
  Reason: no suitable image found.  Did find:
	/Users/mattstyles/projects/node-webkit/node-webkit-builder/example/build/nw-demo/osx64/nw-demo.app/Contents/MacOS/../Frameworks/node-webkit Framework.framework/node-webkit Framework: truncated mach-o error: segment __TEXT extends to 62779392 which is past end of file 65536
	/Users/mattstyles/projects/node-webkit/node-webkit-builder/example/build/nw-demo/osx64/nw-demo.app/Contents/MacOS/../Frameworks/node-webkit Framework.framework/node-webkit Framework: truncated mach-o error: segment __TEXT extends to 62779392 which is past end of file 65536
```

The app is <200kb so I'm guessing the problem is with the nw executable.

Inside `cache/osx64` I have nw, credits.html and the downloaded folder for nw. `cache/osx64/node-webkit` isnt looking too healthy, although `cache/osx64/node-webkit-v0.11.0-osx-x64/node-webkit` is looking good.

So, I'm guessing its not copying across the executable, or not getting it into the right position or something. I tried replacing the broken node-webkit with the good one, but the build just hangs again.

I'm hoping that you're going to tell me just to wait for the ~90mb node-webkit executable to get copied across into the build app, but I've waited a fair time without success, similarly I've cleared the cache folder and waited after the download without success either.

I've got a [repo](https://github.com/mattstyles/nw-build-test) I've been mucking around in which exhibits the same behaviour as using the example app in this repo.

**Comment by mattstyles** [comment]
Did some digging, the hanging after download gets stuck in ncp, specifically 

https://github.com/mllrsohn/node-webkit-builder/blob/master/lib/downloader.js#L175

Not sure if #170 moves to resolve copying files errors but does suggest some problems with ncp.

**Comment by mattstyles** [comment]
I swapped out kopeer for ncp and all works fine again now.

https://github.com/mattstyles/node-webkit-builder

I wont issue a PR as I think I've seen this mentioned elsewhere and I'm not sure what else will break (if anything) by using replacing ncp with kopeer.

**Comment by adam-lynch** [comment]
Ok, there are some known problems around copying (EMFILES, etc.) *But* maybe this is mostly due to an old version of node-webkit-builder in the example directory? See #163.

**Comment by mattstyles** [comment]
Thanks for the response Adam, not sure about old version, I've just grabbed directly from master after (I think) trying with the latest release. I've also tried a couple of different versions of nw too.

In any case, I'm almost certain things like this will be cleared up along with the other fixes for EMFILES etc (I got a boatload of those errors at first too). I'll close this as I dont think its specific enough and I think it'll get fixed anyway. Cheers.
